frontend/components/ClubRank/RankDropdown.tsx

```diff
@@ -0,0 +1,120 @@
+import { ReactElement, useEffect, useState } from 'react'
+import styled, { keyframes } from 'styled-components'
+
+import { Club } from '~/types'
+import { doApiRequest } from '~/utils'
+
+import ClubCard from '../ClubCard'
+import { Collapsible } from '../SearchBar'
+
+type RankDropdownProps = {
+  reloadRankings: () => void
```

Comment:
Make optional, call it `onReloadRankings?`

Reply:
🫡